src/Campaigns/Migrations/Tables/CreateCampaignFormsTable.php

```diff
@@ -50,8 +50,7 @@ public function run(): void
 
         $sql = "CREATE TABLE $table (
             campaign_id INT UNSIGNED NOT NULL,
-            form_id INT UNSIGNED NOT NULL,
-            is_default BOOLEAN NOT NULL DEFAULT 0,
+            form_id INT UNSIGNED NOT NULL
```

Comment:
This is failing automated testing with a SQL syntax error:

```
WordPress database error You have an error in your SQL syntax;
check the manual that corresponds to your MySQL server version
for the right syntax to use at line 4.

CREATE TEMPORARY TABLE wptests_give_campaign_forms (
    campaign_id INT UNSIGNED NOT NULL,
    form_id INT UNSIGNED NOT NULL
    KEY form_id (form_id),
    KEY campaign_id (campaign_id),
    PRIMARY KEY (campaign_id, form_id)
) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_520_ci
```

Reply:
Looks like a missing comma.

```suggestion
            form_id INT UNSIGNED NOT NULL,
```

Reply:
Good catch!